tfx/components/evaluator/executor.py

```diff
@@ -176,9 +180,21 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
           eval_saved_model_path=model_path,
           add_metrics_callbacks=add_metrics_callbacks))
 
-    file_pattern = io_utils.all_files_pattern(
-        artifact_utils.get_split_uri(input_dict[constants.EXAMPLES_KEY], 'eval')
-    )
+    # Load and deserialize examples path splits from execution properties.
+    examples_path_splits = json_utils.loads(
+        exec_properties.get(constants.EXAMPLES_PATH_SPLITS_KEY)) or []
```

Comment:
`json_utils.loads(exec_properties.get(xxx, 'null')) or []` is correct:

exec_properties.get('xxx', 'null') will return 'null' when exec_properties['xxx'] = None
json_utils.loads('null') will return None, then None or [] will be []

Reply:
i understand the difference now, i will correct the other components